documentation/source/intro/overview.rst

```diff
@@ -0,0 +1,79 @@
+.. _Getting-Started:
+
+OpenEBS Introduction
+====================
+OpenEBS is a cloud native storage solution built with the goal of providing containerized storage for containers. Using OpenEBS, a developer can seamlessly get the persistent storage for stateful applications with ease, much of which is automated, while using the popular orchestration platforms such as Kubernetes.
+
+OpenEBS is Container Attached Storage(CAS) - which combines the best of DAS and NAS Storage Solutions. OpenEBS runs within a Kubernetes Cluster as Pods and provides Persistent Volumes to Stateful applications, by making use of storage attached to the Kubernetes Nodes.
+
+.. image:: ../_static/das-nas-cas.png
+    :align: center
+
+You can try out OpenEBS on your Kubernetes Cluster using the `Quick Start Guide`_. 
+ 
+.. _Quick Start Guide: http://openebs.readthedocs.io/en/latest/getting_started/quick_install.html
+
+When you install OpenEBS, you get the following:
+
+* Dynamic Volume Provisioner
+* Storage Classes
+* OpenEBS Operators and Custom Resources for Managing Storage
```

Comment:
It would be good if we could explain these high level concepts? Whats a storage class what does it do? etc?

Reply:
these are k8s concepts. added a link to k8s docs.